optional/fetchmail/fetchmail.py

```diff
@@ -118,8 +100,9 @@ def run(debug):
     os.chmod("/data/fetchids", 0o700)
     os.setgid(id_fetchmail.pw_gid)
     os.setuid(id_fetchmail.pw_uid)
+    system.set_env()
     while True:
-        delay = int(os.environ.get("FETCHMAIL_DELAY", 60))
+        delay = int(os.environ.get("FETCHMAIL_DELAY", 900))
```

Comment:
This is the default in case the variable can't be parsed. We define it to 600 in the default config and that's what should will be used (rather than the fallback value here)